src/frontend/src/test-e2e/views.ts

```diff
@@ -304,39 +280,52 @@ export class MainView extends View {
       .waitForDisplayed({ timeout: 10_000, reverse: true });
   }
 
-  async remove(deviceName: string): Promise<void> {
-    // Open the dropdown by clicking on the trigger button
-    await this.browser
-      .$(`button.c-dropdown__trigger[data-device="${deviceName}"]`)
-      .click();
-    await this.browser
-      .$(`button[data-device="${deviceName}"][data-action='remove']`)
-      .waitForClickable();
-    await this.browser
-      .$(`button[data-device="${deviceName}"][data-action='remove']`)
-      .click();
-  }
-
   async reset(deviceName: string): Promise<void> {
```

Comment:
For consistency and because it is going to bite us sooner or later, I would also add the scrolling here (analogous to `protect` and `unprotect`).

Reply:
@frederikrothenberger done here: https://github.com/dfinity/internet-identity/pull/1656